blockchain/storage/leveldb/batch.go

```diff
@@ -0,0 +1,32 @@
+package leveldb
+
+import (
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+type batchLevelDB struct {
+	db   *leveldb.DB
+	b    *leveldb.Batch
+	Size int
```

Comment:
Do we even need this field? It is just being aggregated in both `Delete` and `Put` functions, but its value is not used anywhere.

```suggestion
```

Reply:
I've also used it when checking the size of the batch. It can be removed.